wagtail/images/tests/test_checks.py

```diff
@@ -0,0 +1,95 @@
+from unittest import TestCase
+import unittest
+from unittest.mock import mock_open, patch
+from wagtail.images.checks import has_jpeg_support
+from wagtail.images import get_image_model
+
+from .utils import Image, get_test_image_file, get_test_image_file_jpeg
+
+import pytest # type: ignore

```

Comment:
Issue: Wagtail doesn't use `pytest` for its tests - please remove (it's not doing anything anyway)